res/css/views/settings/tabs/_SettingsTab.pcss

```diff
@@ -15,7 +15,6 @@ limitations under the License.
 */
 
 .mx_SettingsTab {
-    --SettingsTab_section-margin-bottom-preferences-labs: 30px;
```

Comment:
It is still used here: https://github.com/matrix-org/matrix-react-sdk/blob/dd5560545dfe83c95ab54e8b5492dbcfa99ce47c/res/css/views/settings/tabs/user/_PreferencesUserSettingsTab.pcss#L23

Context: https://github.com/matrix-org/matrix-react-sdk/pull/8828#discussion_r895361529

Reply:
… And should be fine to be removed thanks to https://github.com/matrix-org/matrix-react-sdk/pull/10794/files#diff-f6c132a55221b663c488302b59f34c900cc2e1b0d90e1a4af03053b43fe2adc5L23

Reply:
Ah, missed that when I was splitting this into multiple PRs